### PR TITLE
Add license task.

### PR DESCRIPTION
**Pull request by mattwagl**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,6 +51,7 @@
     "shelljs": "0.7.8"
   },
   "scripts": {
+    "analyze": "eslint --config eslint-config-es/2015/server \"**/*.js\" --ignore-pattern \"test/integration/*/**/*.js\"",
```
Comment (goloroden) [inline package.json]: Shouldn't this be `analyse` to be consistent?

File: configuration/licenseHeuristics.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const licenseHeuristics = {
+  'Apache-2.0*': /Apache License, Version 2\.0/,
+  'GPL-3.0*': /GNU GENERAL PUBLIC LICENSE.*\n.*Version 3,/m,
```
Comment (goloroden) [inline configuration/licenseHeuristics.js]: Is there a special reason why the regular expression does not use `\s*` that we talked about?

File: lib/cli/commands/license.js [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+    if (!options) {
+      throw new Error('Options are missing.');
+    }
+
+    const directory = process.cwd(),
+          { help } = options;
+
+    if (help) {
+      return buntstift.info(getUsage([
+        { header: 'roboter licsense', content: this.description },
```
Comment (goloroden) [inline lib/cli/commands/license.js]: Typo 😉 

(In word `license`, there is an `s` too much)

File: configuration/licenseHeuristics.js (hunk 2 above)
Comment (mattwagl) [reply configuration/licenseHeuristics.js]: It didn't work in the first run but I got it wrong over the phone. I missed the `*` at the end.

File: lib/cli/commands/license.js (hunk 3 above)
Comment (mattwagl) [reply lib/cli/commands/license.js]: Will fix.

File: package.json (hunk 1 above)
Comment (mattwagl) [reply package.json]: Copied it over from the old branch. There we used the term `analyze`. But sure, I can change it.

**Comment by mattwagl** [comment]
I included your feedback. And I also added a proposal for a circleci configuration based an the new V2 yaml format. I was already able to run this config locally using [their new CLI](https://circleci.com/docs/2.0/local-jobs/) which is really handy (just install the CLI and run `circleci build` ). I think the last missing step is to configure the Github repo accordingly. It seems like it's just a problem of access rights. On circle ci your currently get the following output:

```
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
Exited with code 128
```

**Pull request merged**